templates/docs/examples/patterns/rule/dark.html

```diff
@@ -3,11 +3,12 @@
 
 {% block standalone_css %}patterns_rule{% endblock %}
 
+{% set is_dark = True %}
 {% block content %}
 
-<div class="p-strip is-dark">
+<div class="p-strip">
     <div class="u-fixed-width p-section--shallow">
-      <hr class="p-rule--highlight is-dark">
+      <hr class="p-rule--highlight">
```

Comment:
This will also not work, the rule component does not support new theming yet, so it will not inherit the theme. It needs to have `is-dark` class on itself.

So the switch will not work here yet, please revert this example.